packages/cursorless-engine/src/processTargets/modifiers/surroundingPair/index.ts

```diff
@@ -89,7 +89,7 @@ function processSurroundingPairCore(
       return findSurroundingPairTextBased(
         editor,
         range,
-        containingScope.contentRange,
+        containingScope[0].contentRange,
```

Comment:
wasn't sure what to do here, but I don't think it matters too much: it's for legacy surrounding pair modifier, and would be the situation where there are is a text fragment capture that has multiple targets, which I can't really see being a case we'll need to support